src/ext/java/org/opentripplanner/ext/transmodelapi/model/network/GroupOfLinesType.java

```diff
@@ -0,0 +1,46 @@
+package org.opentripplanner.ext.transmodelapi.model.network;
+
+import graphql.Scalars;
+import graphql.schema.GraphQLFieldDefinition;
+import graphql.schema.GraphQLNonNull;
+import graphql.schema.GraphQLObjectType;
+import org.opentripplanner.ext.transmodelapi.mapping.TransitIdMapper;
+import org.opentripplanner.model.GroupOfRoutes;
+
+public class GroupOfLinesType {
+
+    private static final String NAME = "GroupOfLines";
+
+    public static GraphQLObjectType create() {
+        return GraphQLObjectType
+                .newObject()
+                .name(NAME)
+                .description("Additional (optional) grouping of lines for particular purposes such as e.g. fare harmonisation or public presentation.")
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("id")
+                        .type(new GraphQLNonNull(Scalars.GraphQLID))
+                        .dataFetcher(env -> TransitIdMapper.mapEntityIDToApi(env.getSource()))
+                        .build())
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("privateCode")
+                        .type(Scalars.GraphQLString)
+                        .dataFetcher(env -> ((GroupOfRoutes) env.getSource()).getPrivateCode())
+                        .build())
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("shortName")
+                        .type(Scalars.GraphQLString)
+                        .dataFetcher(env -> ((GroupOfRoutes) env.getSource()).getShortName())
+                        .build())
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("name")
+                        .type(Scalars.GraphQLString)
+                        .dataFetcher(env -> ((GroupOfRoutes) env.getSource()).getName())
+                        .build())
+                .field(GraphQLFieldDefinition.newFieldDefinition()
+                        .name("description")
+                        .type(Scalars.GraphQLString)
+                        .dataFetcher(env -> ((GroupOfRoutes) env.getSource()).getDescription())
+                        .build())
+                .build();
+    }
+}
```

Comment:
- You need descriptions for the type and all fields here. 
 - The name is required. 
 - None of the other fields are listed in the Nordic profile: https://enturas.atlassian.net/wiki/spaces/PUBLIC/pages/728563886/network#GroupOfLines -  Are realy. Do you need all of them?

Reply:
They are listed in GroupOfEntities which GroupOfLines is a subset of 
https://enturas.atlassian.net/wiki/spaces/PUBLIC/pages/728727624/framework#framework-GroupOfEntities

Reply:
For me those names are self-explanatory so I do not see need for adding any description for those fields. It would be pretty much the same information as in names. Is there a "rule" that requires all fields to have description? There are a lot of other types that omit having descriptions at the moment.

Reply:
We do not add doc if it is "self-describing", and I know there are a lot of code in the internal model that lack documentation - mainly because the code was copied "as is" into OTP from another project, but we try to add doc at new code at least.

 If you add `@NotNull` to getters for all required fields, that would cover the documentation for req/opt. As mentioned in the meeting yesterday, you may find useful doc in the specifications. 

Hint! I usually ask my self: What is the purpose of this class/method/field? What is it used for? Is there any alternatives/special cases?

The [Codestyle](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/docs/Codestyle.md) have one section on documentation, but it lack useful help for fields.

Reply:
I've added some documentation to GraphQL definition now